pkg/build/api/types.go

```diff
@@ -911,6 +916,9 @@ type GenericWebHookEvent struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
+
+	// StrategyOptions contains additional strategy-specific options for the build
+	StrategyOptions *BuildStrategyOptions
```

Comment:
I used strategy options because it seems like we want to have a generic struct instead of BuildArgs, which is DockerStrategy specific. Should I wait for API review on this, or change now?